plat/unix/update_tags.sh

```diff
@@ -69,9 +69,24 @@ echo $$ > "$TAGS_FILE.lock"
 # Remove lock and temp file if script is stopped unexpectedly.
 trap 'errorcode=$?; rm -f "$TAGS_FILE.lock" "$TAGS_FILE.temp"; exit $errorcode' INT QUIT TERM EXIT
 
+# Change to directory of tags file for relative paths.
+cd "$(dirname "$TAGS_FILE")"
+
 INDEX_WHOLE_PROJECT=1
 if [ -f "$TAGS_FILE" ]; then
     if [ "$UPDATED_SOURCE" != "" ]; then
+        # Make the file path relative to the tags file.
+        # This is required for consistent naming.
+        # See https://github.com/ludovicchabant/vim-gutentags/issues/70.
+        relpath() {
+            python -c "import os.path; print(os.path.relpath('$1', '${2}'))"
```

Comment:
I know only one way so far: `ln --relative --symbolic` with `readlink`. But this is GNU-coreutils-specific, [BSD ln](https://www.freebsd.org/cgi/man.cgi?ln) seems to not have this argument. I am wondering why you want exactly relative path? It seems that absolute path is easier to construct:

```Shell
abspath() {(
    test "x$2" '!=' "x" && cd "$2"
    cd "$(dirname "$1")"
    printf '%s/%s\n' "$(pwd)" "$(basename "$1")"
)}
```